tools/vpn/vpn_softetherbridge.md

```diff
@@ -138,7 +147,7 @@ network:
   version: 2
   ethernets:
     eth0:
-      dhcp4: false <--- Make sure to change to false!!! Otherwise you cannot access your device over ether anymore.
+      dhcp4: false <--- Make sure to change to false!!! Otherwise you cannot access your device over ethernet anymore.
```

Comment:
```suggestion
      dhcp4: false <--- Make sure to change to false!!! Otherwise, you will no longer be able to access your device over Ethernet.
```